components/core/src/os/process/windows.rs

```diff
@@ -32,6 +32,16 @@ pub fn become_command(command: PathBuf, args: Vec<OsString>) -> Result<()> {
     become_child_command(command, args)
 }
 
+/// Get process identifier of calling process.
+pub fn current_pid() -> u32 {
+    unsafe { kernel32::GetCurrentProcessId() as u32 }
+}
+
+/// Determines if a process is running with the given process identifier.
+pub fn is_alive(pid: u32) -> bool {
+    handle_from_pid(pid).is_some()
```

Comment:
Generally good hygene. Here is a quote from the docs: 

> In general, CloseHandle invalidates the specified object handle, decrements the object's handle count, and performs object retention checks. After the last handle to an object is closed, the object is removed from the system.

The docs on `OpenProcess` state this should be done. The `child` struct does this in a `drop`.